pkg/apps/blackduck/sizes/v1/size.go

```diff
@@ -164,13 +165,502 @@ func (s size) GetSize(name string) map[string]*types.Size {
+				},
+			},
+			"cfssl": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.CfsslContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 640,
+						MaxMem: 640,
+					},
+				},
+			},
+			"documentation": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.DocumentationContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 512,
+						MaxMem: 512,
+					},
+				},
+			},
+			"jobrunner": {
+				Replica: 6,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.JobrunnerContainerName: {
+						MinCPU: 1,
+						MaxCPU: 1,
+						MinMem: 13824,
+						MaxMem: 13824,
+					},
+				},
+			},
+			"rabbitmq": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.RabbitMQContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 1024,
+						MaxMem: 1024,
+					},
+				},
+			},
+			"registration": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.RegistrationContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 1024,
+						MaxMem: 1024,
+					},
+				},
+			},
+			"scan": {
+				Replica: 3,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.ScanContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 9728,
+						MaxMem: 9728,
+					},
+				},
+			},
+			"solr": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.SolrContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 640,
+						MaxMem: 640,
+					},
+				},
+			},
+			"uploadcache": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.UploadCacheContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 512,
+						MaxMem: 512,
+					},
+				},
+			},
+			"webapp-logstash": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.WebappContainerName: {
+						MinCPU: 2,
+						MaxCPU: 2,
+						MinMem: 9728,
+						MaxMem: 9728,
+					},
+					types.LogstashContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 1024,
+						MaxMem: 1024,
+					},
+				},
+			},
+			"webserver": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.WebserverContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 2048,
+						MaxMem: 2048,
+					},
+				},
+			},
+			"zookeeper": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.ZookeeperContainerName: {
+						MinCPU: 0,
+						MaxCPU: 0,
+						MinMem: 640,
+						MaxMem: 640,
+					},
+				},
+			},
+			"postgres": {
+				Replica: 1,
+				Containers: map[types.ContainerName]types.ContainerSize{
+					types.PostgresContainerName: {
+						MinCPU: 2,
+						MaxCPU: 2,
+						MinMem: 12288,
+						MaxMem: 12288,
+					},
+				},
+			},
+		}
+	case "XLARGE":
```

Comment:
This is `X-LARGE` in all other places: https://github.com/blackducksoftware/synopsys-operator/search?q=x-large&unscoped_q=x-large